src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] result = new int[2];
```

Comment:
let's create 2 variables with informative names instead of this array

Reply:
```suggestion
        int totalBuy = 0;
        int totalSupply = 0;
```
and use them in your code
totalBuy is better naming than result[BUY_INDEX]